explainer.md

```diff
@@ -532,26 +533,40 @@ interface VRSession : EventTarget {
   attribute EventHandler onfocus;
   attribute EventHandler onresetpose;
 
-  VRSourceProperties getSourceProperties(optional double scale);
+  readonly attribute VRDevicePose pose;
 
   Promise<VRFrameOfReference> createFrameOfReference(VRFrameOfReferenceType type);
-  VRDevicePose? getDevicePose(VRCoordinateSystem coordinateSystem);
 
+  Promise<VRPresentationFrame> requestVRPresentationFrame();
+  
   Promise<void> endSession();
 };
 
 //
-// Pose
+// Presentation Frame, Device Pose, and Views
 //
+interface VRPresentationFrame {
+  readonly attribute FrozenArray<VRView> views;
 
-interface VRDevicePose {
-  readonly attribute Float32Array leftProjectionMatrix;
-  readonly attribute Float32Array leftViewMatrix;
+  VRDevicePose? getDevicePose(VRCoordinateSystem coordinateSystem);
+
+  VRWebGLViewport getViewport(VRWebGLLayer layer, VRView? view);
```

Comment:
@travisleithead suggested that the overload model was applicable here. Though an alternative could be (type name feedback welcome!):
```js
interface VRLayerViewTarget {
};

interface VRWebGLLayerViewTarget : VRLayerViewTarget  {
  readonly attribute int viewportX;
  readonly attribute int viewportY;
  readonly attribute int viewportWidth;
  readonly attribute int viewportHeight;
};

partial interface VRPresentationFrame {
  VRLayerViewTarget getLayerViewTarget(VRLayer layer, VRView? view);
};
```